ci/trigger-release-branch-ci.rs

```diff
@@ -0,0 +1,67 @@
+//! Helper script used by `.github/workflows/ci-cron-trigger.yml`
+
+use std::process::Command;
+
+fn main() {
+    let output = Command::new("git")
+        .arg("for-each-ref")
+        .arg("refs/remotes/origin")
+        .arg("--format")
+        .arg("%(refname)")
+        .output()
+        .unwrap();
+    assert!(output.status.success());
+    let mut releases = std::str::from_utf8(&output.stdout)
+        .unwrap()
+        .lines()
+        .filter_map(|l| l.strip_prefix("refs/remotes/origin/release-"))
+        .filter_map(|l| {
+            let mut parts = l.split('.');
+            let major = parts.next()?.parse::<u32>().ok()?;
+            let minor = parts.next()?.parse::<u32>().ok()?;
+            let patch = parts.next()?.parse::<u32>().ok()?;
+            Some((major, minor, patch))
+        })
+        .collect::<Vec<_>>();
+    releases.sort();
+
+    let mut to_trigger: Vec<(u32, u32, u32)> = Vec::new();
+    let mut iter = releases.iter().rev();
+
+    // Pick the latest 3 release branches to keep up-to-date. Although we
+    // only promise the last 2 are going to be released with security fixes when
+    // a new release branch is made that means there's one "pending" release
+    // branch and two "active" release branches. In that situation we want to
+    // update 3 branches. If there's no "pending" branch then we'll just be
+    // keeping some older branch's CI working, which shouldn't be too hard.
+    to_trigger.extend(iter.by_ref().take(3));
+
+    // We support two LTS channels 12 versions apart. If one is already included
+    // in the above set of 3 latest releases, however, then we're only picking
+    // one historical LTS release.
+    let mut lts_channels = 2;
+    if to_trigger.iter().any(|(major, _, _)| *major % 12 == 0) {
+        lts_channels -= 1;
+    }
```

Comment:
This is technically somewhat inaccurate. If an LTS branch has been created, but not released, then we need to still look back 2 releases. The risk of that causing issues is pretty small though as that's a 2-week window and I otherwise don't know how to easily test for whether the latest branch has been released or not as part of this script.